packages/origin-247-certificate/src/offchain-certificate/offchain-certificate.service.ts

```diff
@@ -202,6 +224,7 @@ export class OffChainCertificateService<T = null> {
                 });
                 await this.createAggregate(events);
             }
+            await claimBatchCommandValidator(commands);
```

Comment:
And here, before creating aggregate or anything, this should be first thing in the code